### DISCUSSION TITLE
How to Change Favicon?

### DISCUSSION THREAD

**Discussion by chicagocomputerclasses** [fireactjs/core#3] [Q&A]
I'm trying to change favicon to a png file. I've updated the manifest file 
"icons": [
    {
      "src": "favicon.png",
      "sizes": "64x64 32x32 16x16",
      "type": "image/png"
    }
]
Yet it's still grabbing the favicon.ico file.
What am I missing?

**Comment by chaoming** [comment]
This part should be outside of the framework, as part of your Reactjs app. You might just need a full refresh in your browser.
